Foundation/Summit/2024/program.html

```diff
@@ -0,0 +1,373 @@
+---
+#    Copyright 2024 seL4 Project a Series of LF Projects, LLC.
+#    SPDX-License-Identifier: CC-BY-SA-4.0
+title: Summit program
+---
+
+<h1>
+  seL4 Summit 2024
+</h1>
+<h4>
+  Sydney, Australia
+</h4>
+<h4>
+  15-17 October 2024
+</h4>
+
+<h3>
+   Program
+</h3>
+<p>
+  All times are local time in Sydney, Australia (GMT+11).
```

Comment:
Maybe just add, in text, the small paragraph about social activities on Monday and bush walk on weekend.